packages/camera/camera/example/lib/main.dart

```diff
@@ -231,7 +232,14 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
                 ? Container()
                 : SizedBox(
                     child: (localVideoController == null)
-                        ? Image.file(File(imageFile!.path))
+                        ? (
```

Comment:
@ditman Should we include that change in another PR for the camera example?

Reply:
You know it :P